src/components/main-view.tsx

```diff
@@ -6,38 +6,53 @@ import { ForceArrows } from "./force-arrows";
 import "./main-view.scss";
 
 interface IProps {
+  showForce: boolean;
+  version: string;
   saveToTable: (object1: SysObject, object2: SysObject) => void;
   clearTable: () => void;
+  handleChangeShowForceState: (state: boolean) => void;
 }
 
-export const MainView: React.FC<IProps> = ({ saveToTable, clearTable }) => {
+export const MainView: React.FC<IProps> = ({ showForce, version, saveToTable, clearTable, handleChangeShowForceState }) => {
   const [ object1, setObject1 ] = useState<SysObject>(SysObject.bottle);
   const [ object2, setObject2 ] = useState<SysObject>(SysObject.earth);
 
   const handleSave = () => {
     saveToTable(object1, object2);
+    handleChangeShowForceState(true);
+  };
+
+  const showAlert = () => {
+    alert("Click on Calculate forces button to see a representation of the forces between the two objects");
   };
 
   return (
     <div className="main-view">
       <div className="header">Select two objects:</div>
       <div className="columns">
         <div className="column">
-          <div>Object 1: <SelectObject value={object1} onChange={setObject1}/></div>
+          <div data-testid="object-1">Object 1: <SelectObject value={object1} onChange={setObject1} changeShowForceState={handleChangeShowForceState}/></div>
```

Comment:
`handleChangeShowForceState(false)` could also be called in `onChange` handler. Something like:
```
... onChange={handleSetObject1} />
(...)
const handleSetObject1 = (value: string) => {
  setObject1(value);
  handleChangeShowForceState(false);
}
```

We wouldn't have to pass down the handler and make `SelectObject` pulldown to be aware of any logic related to some forces rendered by some parent components (so something totally not related to its own content).

This doesn't matter in this small app, so I'm not asking to change it. It's fine as it is. Just a theoretical comment that could be more useful in bigger apps where this can make a real difference. :wink:

Reply:
Made the suggested changes since it makes more sense.